core/src/main/java/org/jdbi/v3/core/argument/BuiltInArgumentFactory.java

```diff
@@ -53,4 +55,12 @@
             .flatMap(factory -> JdbiOptionals.stream(factory.build(expectedType, value, config)))
             .findFirst();
     }
+
+    private static class LegacyEnumByNameArgumentFactory implements ArgumentFactory {
+        private final EnumArgumentFactory delegate = new EnumArgumentFactory();
+        @Override
+        public Optional<Argument> build(Type expectedType, Object rawValue, ConfigRegistry config) {
+            return delegate.build(QualifiedType.of(expectedType).with(EnumByName.class), rawValue, config);
```

Comment:
:clap:

Reply:
nice, hadn't thought of that simple fix for some reason

Reply:
![](https://cultofthepartyparrot.com/parrots/parrot.gif)